hasura/metadata/actions.yaml

```diff
@@ -106,6 +106,25 @@ actions:
       - role: api_key
       - role: user
     comment: Generates a Signed URL to upload images
+  - name: verify_app
+    definition:
+      kind: synchronous
+      handler: '{{NEXT_API_URL}}/images/_verify_app'
+      headers:
+        - name: Authorization
+          value_from_env: INTERNAL_ENDPOINTS_SECRET
+      request_transform:
+        method: POST
+        query_params:
```

Comment:
will this transform to send the parameters in the body?

Reply:
yes this is generated from the console they just label it as query param but the fields themselves are listed in body.input